extract.go

```diff
@@ -79,6 +79,8 @@ func handleTar(writeCloser io.WriteCloser, readerMaker ReaderMaker, crypter Cryp
 			return errors.Wrap(err, "ExtractAll: tar extract failed")
 		}
 	} else if readerMaker.Format() == "nop" {
+	} else if readerMaker.Format() == "lzo" {
```

Comment:
I'm curious what you think about this patch, which adapts the `Decompressors` slice to make the use of "format not supported" consistent to lzo, but with a slightly different error message, rather than hitting the `lzo_disabled.go` functions that quit immediately.

Reply:
I think this particular branch is the appropriate place. I do not like panic: it scares users by the stack trace and they miss actual error message.

I see here some other problem: we silently ignore unknown files. It is not OK, someone stores something among backup files. At least it is suspicious.

Reply:
I've gone ahead and squashed this in, then.